File: server/ViewModels/Account/AccountViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Area.ViewModels.Account
{
    public class AccountViewModel : SuccessViewModel
    {
        public string Token { get; set; }
        public string Username { get; set; }
    }
}

File: mobile_user/app/src/main/java/area/epitech/area/ViewModels/Account/AccountViewModel.kt
package area.epitech.area.ViewModels.Account

import area.epitech.area.ViewModels.ResultViewModel
import com.github.kittinunf.fuel.core.ResponseDeserializable
import com.google.gson.Gson
import java.io.Reader

class AccountViewModel : ResultViewModel() {
    public val token = ""
    public val username = ""

    class Deserializer : ResponseDeserializable<AccountViewModel> {
        override fun deserialize(reader: Reader) = Gson().fromJson(reader, AccountViewModel::class.java)!!
    }
}